### DISCUSSION TITLE
Can I show fiber with vtk?

### DISCUSSION THREAD

**Discussion by timeanddoctor** [dipy/dipy#2455] [Tractography / Tractometry] [answered]
Can I show fiber  with vtk after tracking?

**Comment by skoudoro** [comment, marked as answer]
Yes, use the function `save_vtk_streamlines` to save it in "*.vtk". you can import it like this: `from dipy.io.vtk import save_vtk_streamlines`.

You can also use [fury package](https://github.com/fury-gl/fury):

```python
streamlines = Streamlines(streamline_generator)

from fury import actor, window
st = actor.streamtube(streamlines)
scene = window.Scene()
scene.add(st)
window.show(scene)
```
